From aa083aa9624ea7b764d5a81c4c676719a1a3e42b Mon Sep 17 00:00:00 2001
From: "djm@openbsd.org" <djm@openbsd.org>
Date: Thu, 20 Sep 2018 03:31:49 +0000
Subject: [PATCH 041/294] upstream: fix "ssh -Q sig" to show correct signature
 algorithm list

(it was erroneously showing certificate algorithms); prompted by markus@

OpenBSD-Commit-ID: 1cdee002f2f0c21456979deeb887fc889afb154d
---
 ssh.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ssh.c b/ssh.c
index 1101ab2..34301c3 100644
--- a/ssh.c
+++ b/ssh.c
@@ -750,7 +750,7 @@ main(int ac, char **av)
 			else if (strcmp(optarg, "key-cert") == 0)
 				cp = sshkey_alg_list(1, 0, 0, '\n');
 			else if (strcmp(optarg, "key-plain") == 0)
-				cp = sshkey_alg_list(0, 1, 0, '\n');
+				cp = sshkey_alg_list(0, 1, 1, '\n');
 			else if (strcmp(optarg, "protocol-version") == 0) {
 				cp = xstrdup("2");
 			}
-- 
1.8.3.1

